src/emails/UserCreatedNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import EmailContent from "types/EmailContent"
+import UserDetails from "types/UserDetails"
+import { UserGroupResult } from "types/UserGroup"
+
+interface Props {
+  user: UserDetails
+}
+
+function printGroups(groups: UserGroupResult[]): string {
+  return groups
+    .map((group: UserGroupResult) => group.friendly_name)
+    .join(", ")
+    .trimEnd()
+}
+
+const UserCreatedNotificationText = ({ user }: Props): string =>
+  `This is an automated message notifying you of the following Bichard account creation.
+  
+            First name: ${user.forenames}
+             Last name: ${user.surname}
+                 Email: ${user.emailAddress}
+              Endorser: ${user.endorsedBy}
+    Permissions Groups: ${printGroups(user.groups)}
+  `
+
+export default function generateUserCreatedNotification(props: Props): EmailContent {
+  return {
+    subject: "Bichard: new user account created",
+    html: "",
```

Comment:
Could we make the `html` attribute optional in our `EmailContent` type, and then not pass it here?

Reply:
Review comment 5